shift_allocation/tests/test_get_allocation_locations.py

```diff
@@ -0,0 +1,188 @@
+from openerp.tests.common import SingleTransactionCase
```

Comment:
As discussed in person I no longer add the encoding declaration because it defaults to UTF-8 and the license statement was only added by Neova Health, we've never added those or modified or removed any existing ones.